mesheryctl/pkg/utils/platform.go

```diff
@@ -768,3 +769,16 @@ func InstallprereqDocker() error {
 	log.Info("Prerequisite Docker Compose is installed.")
 	return nil
 }
+
+// Sets the path to user's kubeconfig file into global variables
+func SetKubeConfig() {
```

Comment:
Moving the logic to common utils code makes sense. Assuming .meshery folder is created already at this point of time.

Reply:
Yes. mesheryctl will create the `.meshery` folder and a default meshconfig file if it is missing while running **any** command.